### DISCUSSION TITLE
PGP密钥对生成器必填项显示问题

### DISCUSSION THREAD

**Discussion by GOOD-AN** [he3-app/he3-feedback#286] [Bug Report]
### Version
pgp-key-pair-generator 1.0.4

### Environment
Windows v1.3.23

### Steps to reproduce
在用户名、电子邮箱与密码填写位置输入后，依旧显示请输入

### What is expected?
输入值后应该不显示请输入提示
只有在必填项为空时才显示提示

### What is actually happening?
无论是否输入都会显示请输入提示
![image](https://github.com/he3-app/he3-feedback/assets/53306377/8c512b4b-84a7-4212-91c1-f1243dd13af0)

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建了 issues 跟进：https://github.com/he3-app/he3-feedback/issues/287

**Reply by GOOD-AN** [reply]
感谢
